### PR TITLE
add eks-config& infrasetup cloudformationtemplets

### PR DESCRIPTION
**Pull request by bharathmadvar**
*k8s infra setup CFT
*slave configuration cft
*vpc config cft
*

### CODE REVIEW

File: aws-prometheus/eks_vpc-confg.yaml [hunk 1]
```diff
@@ -0,0 +1,151 @@
+---
+AWSTemplateFormatVersion: '2010-09-09'
+Description: 'AWS EKS-vpc setup'
+
+Parameters:
+
+  VpcBlock:
+    Type: String
```
Comment (ajlevinson) [inline aws-prometheus/eks_vpc-confg.yaml]: I'd prefer that we automate the Subnetting of the primary CIDR block of the VPC using the Fn:Cidr function... also, why wouldn't we use a Class A CIDR block (e.g. 10.x.x.x) versus a Class C?

File: aws-prometheus/eks_vpc-confg.yaml [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+  ControlPlaneSecurityGroup:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      GroupDescription: Cluster communication with worker nodes
+      VpcId: !Ref VPC
+
+Outputs:
+
+  SubnetIds:
```
Comment (ajlevinson) [inline aws-prometheus/eks_vpc-confg.yaml]: Again, export any / all resources provisioned please.

File: aws-prometheus/k8s-dashboard_setup/deploy-dashboard.yaml [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+---
+#Dashboard Deployment
+
+kind: Deployment
+apiVersion: apps/v1beta2
+metadata:
+  labels:
+    k8s-app: kubernetes-dashboard
+  name: kubernetes-dashboard
+  namespace: kube-system
```
Comment (ajlevinson) [inline aws-prometheus/k8s-dashboard_setup/deploy-dashboard.yaml]: Should the namespace be a parameter versus hard-coded?

Comment (bharathmadvar) [reply aws-prometheus/k8s-dashboard_setup/deploy-dashboard.yaml]: In k8s namespace indicate the "objects created by the Kubernetes system".
https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/#working-with-namespaces
example :
kubectl get namespace
NAME          STATUS    AGE
default       Active    1d
kube-system   Active    1d
kube-public   Active    1d

**Pull request merged**